### ISSUE TITLE
Feature Request : differnet modems on roms

### ISSUE THREAD

**Issue by nik012003** [Tasssadar/multirom#41]
It is possibie to add the function to have several modems on Roms secondary ?

**Comment by Jens14L** [comment]
No, I think not … The radio as well as the bootloader software is firmware and is directly flashed in the bootloader/download mode.
